processor/processor.go

```diff
@@ -82,6 +83,7 @@ type Handle struct {
 
 	logger                    logger.Logger
 	eventSchemaHandler        types.EventSchemasI
+	enricher                  enricher.PipelineEnricher
```

Comment:
Don't we need a list of enrichers instead?

Reply:
We can always pass an enricher which chains other enrichers or have an array passed here. Given we started with this change and requirements for next enricher aren't much clear, I left it at that only. Wdyt ?

Reply:
What's the point of defining a generic enricher interface if processor knows implementation details, i.e. that this is a geoenricher. Using a list of enrichers simplifies the case where no enricher is enabled too :)

Reply:
I agree. I have made the updates now.